src/shared/SellerProfile/ReviewHeader.js

```diff
@@ -26,14 +28,14 @@ const ReviewHeader = (props) => {
   };
 
   return (
-    <section className="seller-profile seller-profile__review-header">
+    <section styleName="profile.review-header">
       <div className="row ">
         <div className="col-sm-8">
-          <h1 tabIndex="-1">{name}</h1>
+          <h1 tabIndex="-1" styleName="profile.heading">{name}</h1>
```

Comment:
shouldn't this be styleName={profile.heading} ? or that's the syntax from loader you're using?